### PR TITLE
feat: Optionally collect machine specs via SSH

### PR DESCRIPTION
**Pull request by YairHalberstadt**
If user specifies vmUserName we attempt to collect machine specs via SSH if the DB is remote.

In order to avoid duplication we also replace mysql/db-machine-specs.sh with a symbolic link. When packaging we reify the symbolic link.

### CODE REVIEW

**Comment by YairHalberstadt** [comment]
cc @liozme

File: scripts/collector/postgres/db-machine-specs.sh [hunk 1]
```diff
@@ -41,30 +42,44 @@ echo "$headers" > "$outputPath"
...
-totalSizeBytes=$(df --total / | awk '/total/{printf("%.0f\n", ($2+0) * 1024)}')
-usedSizeBytes=$(df --output=used -B1 / | awk 'NR==2{printf("%.0f\n", ($1+0))}')
+
+# Check if machineName is "localhost"
+if [ "$machine_name" = "0.0.0.0" ] || grep -q "$machine_name" /etc/hosts; then
+    source <(echo "$coreScript")
+else
+    if [[ -z "$userName" ]]; then
+        echo "VM User name not set, skipping."
+        exit 0
```
Comment (liozme) [inline scripts/collector/postgres/db-machine-specs.sh:65]: even in case of failure, i would like to have the CSV populated with the hostname and zeroes.

Comment (liozme) [reply scripts/collector/postgres/db-machine-specs.sh:65]: the hostname is the more important part it as we use it

File: scripts/collector/postgres/collect-data.sh [hunk 2]
```diff
@@ -519,7 +532,10 @@ manualUniqueId=""
 	 elif [[ "$1" == "--collectionUserName" ]]; then collectionUserName="${2}"
 	 elif [[ "$1" == "--collectionUserPass" ]]; then collectionUserPass="${2}"
 	 elif [[ "$1" == "--connectionStr" ]];      then connStr="${2}"
-	 elif [[ "$1" == "--manualUniqueId" ]];      then manualUniqueId="${2}"
+	 elif [[ "$1" == "--manualUniqueId" ]];     then manualUniqueId="${2}"
+   elif [[ "$1" == "--vmUserName" ]];         then vmUserName="${2}"
```
Comment (liozme) [inline scripts/collector/postgres/collect-data.sh:536]: spacing

File: scripts/collector/postgres/db-machine-specs.sh (hunk 1 above)
Comment (YairHalberstadt) [reply scripts/collector/postgres/db-machine-specs.sh:66]: Done

**Review by cofin — APPROVED** [review]
lgtm

**Pull request merged**